src/plonemeeting/portal/core/configure.zcml

```diff
@@ -159,6 +160,19 @@
       description="Fetched data from delib are kept if used AND URL parameters are now in a datagridfiel widget"
       handler="plonemeeting.portal.core.migrations.migrate_to_1006.migrate" />
   </genericsetup:upgradeSteps>
+
+  <genericsetup:upgradeSteps
+      source="1006"
+      destination="1007"
+      profile="plonemeeting.portal.core:default">
+    <genericsetup:upgradeDepends
+      title="Re-apply registry"
```

Comment:
Loin d'être bloquant mais j'aime bien mettre du contexte dans les title et autre description çà sert tjs à un moment, soit à celui qui a fait le code, ou qq'un d'autre qui débarque, par exemple "Re-apply registry to add GDPR masked text related attributes" par exemple ;-)  Sinon çà me semble bon tu peux merger!